lib/osf-components/addon/components/file-browser/file-rename-modal/component.ts

```diff
@@ -32,7 +32,12 @@ export default class FileRenameModal extends Component<Args> {
 
     @action
     resetFileNameValue() {
-        this.newFileName =  this.originalFileName;
+        return this.newFileName =  this.originalFileName;
+    }
+
+    @action
+    updateOriginalName(newName: string) {
+        return this.originalFileName =  newName;
```

Comment:
Don't need an action to achieve this. Simply do `this.originalFileName = newName` in the task below is sufficient.

Reply:
Got it, thank you Yuhuai